casa/Containers/test/tBlock.cc

```diff
@@ -42,6 +42,36 @@
 
 #include <casacore/casa/namespace.h>
```

Comment:
this file needs a `stdint.h` (or cstdin for c++11) include for `intptr_t´ otherwise it fails on some platforms (e.g. ubuntu 14.04)